install/autodetect.go

```diff
@@ -128,9 +129,20 @@ func (k *K8sInstaller) autodetect(ctx context.Context) {
 	}
 }
 
-func (k *K8sInstaller) autodetectAndValidate(ctx context.Context) error {
-	k.autodetect(ctx)
+func getClusterName(helmValues map[string]interface{}) string {
+	cluster, ok := helmValues["cluster"].(map[string]interface{})
+	if !ok {
+		return ""
+	}
+	chainingMode, ok := cluster["name"].(string)
```

Comment:
`chainingMode` here should be `name`

Reply:
this is what happens when you copy and paste functions